src/details/ArborX_DetailsTreeConstruction.hpp

```diff
@@ -69,7 +68,7 @@ inline void calculateBoundingBoxOfTheScene(ExecutionSpace const &space,
   using Access = AccessTraits<Primitives, PrimitivesTag>;
   auto const n = Access::size(primitives);
   Kokkos::parallel_reduce(
-      ARBORX_MARK_REGION("calculate_bounding_box_of_the_scene"),
+      "ArborX::TreeConstruction::calculate_bounding_box_of_the_scene",
```

Comment:
Maybe
```suggestion
      "ArborX::BVH::TreeConstruction::calculate_bounding_box_of_the_scene",
```
but don't feel strong about it.

Reply:
Well this very kernel could technically be used by the brute force impl